Storage/Storage/Model/WooCommerce.xcdatamodeld/Model 27.xcdatamodel/contents

```diff
@@ -0,0 +1,554 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="15702" systemVersion="19D76" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="Account" representedClassName="Account" syncable="YES">
```

Comment:
_This is a discussion for the file itself, not line 3_. 

If we change the relationships based on my discussion above, we'd probably also need to do a custom migration. Using Model 26, our database rows for categories look like this:

![image](https://user-images.githubusercontent.com/198826/78410179-d541f500-75c8-11ea-94dd-7362323d7155.png)

Notice that we have a lot of duplicates. We'll probably have to consolidate that if the product → categories relationship is no longer `Cascade Delete`.

Reply:
I guess this is because each product had its own category as they weren't shared, right? 

I wonder if it's good enough to delete duplicated-orphan categories when doing `SynchonizeCategories` action? 🤔

Reply:
I'd rather not have a "purge" code that is active all the time. I think it's better if we have some sort of migration that only happens when the user has upgraded the app. 

Would you mind creating an issue for the plan for the migration? And then we can decide the priority for it. It can probably be done later after the UI changes as long as we don't forget to remove orphaned records.